avocado/utils/service.py

```diff
@@ -267,6 +267,8 @@ def sys_v_init_command_generator(command):
     elif command == 'disable':
         command_name = "chkconfig"
         command = "off"
+    elif command == "raw_status":
```

Comment:
what is the `raw_status` command? I don't know it and I can't find it anywhere...

Reply:
Actually it maps to "status" on both. What is the purpose of this, it looks duplicate and unnecessary...

Reply:
This is from https://github.com/avocado-framework/avocado-vt/blob/master/virttest/staging/service.py . It prints out complete status message of a process. I did not complete implementation. But it seems redundant here, so I will remove it.